src/common/plugin/DatabasePluginManager.C

```diff
@@ -52,11 +53,19 @@ DatabasePluginManager::DatabasePluginManager() : PluginManager("database")
 //    Brad Whitlock, Wed Jun 25 10:27:17 PDT 2008
 //    Call UnloadPlugins here since it calls virtual methods for this class.
 //
+//    Kathleen Biagas, Thu Mar 11 2021
+//    On Windows, only unload plugins when debugging memory leaks.
+//    There is a crash during the unloading of the plugins at exit, causing
+//    mdserver memory dump files to be silently generated, sometimes even
+//    resulting in a Windows error dialog popping up on exit.
+//
 // ****************************************************************************
 
 DatabasePluginManager::~DatabasePluginManager()
 {
+#if !defined(_WIN32 ) || defined(DEBUG_MEMORY_LEAKS)
```

Comment:
Should this be "&&" and not "||". As it stands, I think if we were on Linux, the if would be true and we would unload the plugins.

Reply:
The intent was not to change the behavior on linux or Mac.

Reply:
If you think the behavior should be changed on all platforms, I can just remove the _WIN32 part, and then we will only unload the plugins when debugging leaks.

Reply:
Yes, I think only unloading plugins when debugging leaks is the safest. There is no reason to unload the plugins when exiting and it can only potentially cause problems (unless of course looking for memory leaks).

Reply:
Made the change.